crawl-ref/source/dat/strings/de/hiscores.txt

```diff
@@ -636,10 +636,10 @@ Tarried too long and was consumed by Zot
 Hat zu lange gezögert und wurde von Zot verschlungen
 %%%%
 constriction
-Zerquetschung
+Umschlingung
 %%%%
 Constricted to death by %s
-Von %s zu Tode zerquetscht
+Von %s zu Tode umschlungen
```

Comment:
Here, 'zu Tode gedrückt' would be better. Or just: 'Von %s zerquetscht'

Reply:
OK. I will change to gedrückt because I've used zerquetscht for squashed.

Reply:
done